File: Getting Started/16.cpp
//Fibonacci Series upto nth term
#include <iostream>
using namespace std;
int main(){
    cout<<"Enter the number of terms: ";
    int n;
    cin>>n;
    int a=0, b=1;
    for(int i=0; i<n; i++){
        cout<<a<<" ";
        int c=a+b;
        a=b;
        b=c;
    }
    return 0;
}

File: Getting Started/16.py
##fibonacci series upto n terms

n = int(input("Enter the number of terms: "))
a = 0
b = 1
print(a, b, end = " ")
for i in range(2, n):
    c = a + b
    print(c, end = " ")
    a = b
    b = c

